lib/Cro/HTTP/Request.pm6

```diff
@@ -83,6 +83,7 @@ class Cro::HTTP::Request does Cro::HTTP::Message {
         die X::Cro::HTTP::Request::Incomplete.new(:missing<method>) unless $!method;
         die X::Cro::HTTP::Request::Incomplete.new(:missing<target>) unless $!target;
         my $version = self.http-version // (self.has-header('Host') ?? '1.1' !! '1.0');
+        $version = '1.1' if $version eq '2.0';
```

Comment:
Hmm, wonder if better would be to add a `:$force-version` named parameter, so we don't lose the "debug dump" value of it showing 2.0 when that's where it originally came from?

Reply:
IMO that's a bad idea.

To differentiate between HTTP versions we already show underlying HTTP/2 frames in the debug output, which is enough to judge. But when we show outright violation of the protocol in the debug output that's IMO much more confusing.